### PR TITLE
[STU-285] Adding support for optional upload to cloud

### PR DESCRIPTION
**Pull request by LatentDream**
Blocked by https://github.com/flojoy-ai/studio/pull/1123
Once merge, add the new exportToCloud to the createNewTest function

### CODE REVIEW

**Comment by LatentDream** [comment]
Blocked by [project PR](https://github.com/flojoy-ai/studio/pull/1123)
Once merge, add the new exportToCloud to the `createNewTest` function

File: captain/utils/test_sequencer/run_test_sequence.py
Comment (izi-on) [inline captain/utils/test_sequencer/run_test_sequence.py:294]: use the wrapper instead (refer to _case_pytest or any of those) , create new function for this that is wrapped with it

File: captain/utils/test_sequencer/run_test_sequence.py
Comment (izi-on) [inline captain/utils/test_sequencer/run_test_sequence.py:292]: what if we have two functions with the same name at the end but different scope?

**Review by izi-on — CHANGES_REQUESTED** [review]
small things

File: captain/utils/test_sequencer/run_test_sequence.py
Comment (LatentDream) [reply captain/utils/test_sequencer/run_test_sequence.py:292]: good point :/ Just weird to have the file_name::name.
We really need a rename test functionnality

Comment (LatentDream) [reply captain/utils/test_sequencer/run_test_sequence.py]: Right now in the cloud, the test name is:
Downloads/project/test.py::test_one

I'll add a rename test soon in the IU!

**Review by izi-on — APPROVED** [review]
lgtm

**Pull request merged**
